### DISCUSSION TITLE
FA617NT - AMD Advantage ASUS TUF 16 Inch App Performance (0.161.0)

### DISCUSSION THREAD

**Discussion by the1corrupted** [seerge/g-helper#2330] [General]
Seerge, first thank you for providing this app. I initially installed 0.161.0 and every feature appeared to work excellently.

This GHelper has also made other settings WAY easier to find than in Armory crate, and that actually matter to me. Like CPU package power, limiting mine from 80W to 70W to see what kind of impact that might have.
It detected the display at 165 hz correctly the first time.

"Smart Charging" was way easier to find and manage than in the BIOS or buried somewhere in Armoury Crate UI. 99% of the time, this laptop is remaining plugged in (because I game, duh).

I left a donation, and I hope to see a full version of this app available for everyone because of just how badly ASUS's software UI really is. It's stuck in the early 2000's and has no intuition at all in terms of design.

Something _a bit_ confusing is the power limit sliders with a check box to "Apply power limits" but what is the behavior otherwise?

**Comment by seerge** [comment]
@the1corrupted hello, thanks for support

>Something a bit confusing is the power limit sliders with a check box to "Apply power limits" but what is the behavior otherwise?

If ``Apply`` is not checked, system will use whatever power limits Asus has hardcoded for each of the modes (Silent / Balanced / Turbo) by default. It varies from model to model. There is no way to "read" default values, that's why Apply exists in a first place

**Reply by the1corrupted** [reply]
Something else has revealed itself when I was configuring power limits, and playing with some power settings.

I had an issue where I replaced the factory SSD in the laptop with a brand new SSD but the new SSD was getting WAY too hot under gaming loads. I would see the SMART temperature reach above 70, and sometimes 80.

I found an aftermarket copper heatsink to place on the SSD, and while this helped, it was not the entire solution.

Now that I can limit the CPU package power, this seems to have also decreased the sustained SSD temperature under gaming load!

I went from 70+ sustained temperatures to a range of 62-68 when limit CPU power to just 50W. I am also seeing more consistent performance in my games with the reduced CPU power.

The games I tend to play don't need a ton of compute power, and I feel like if I ever have a problem in the future, I can just raise the limit.

Just "happened" to fix another temperature issue I was having and I can't thank you enough.
![image](https://github.com/seerge/g-helper/assets/30715215/3baf8310-932d-4cd5-903e-d6a1dae1d3ee)

**Reply by seerge** [reply]
Great, from my observations most games run absolutely same even with 35-40W limit.
